src/trackers/COMMON.py

```diff
@@ -23,6 +23,7 @@ async def edit_torrent(self, meta, tracker, source_flag, torrent_filename="BASE"
                     new_torrent.metainfo.pop(each, None)
             new_torrent.metainfo['announce'] = self.config['TRACKERS'][tracker].get('announce_url', "https://fake.tracker").strip()
             new_torrent.metainfo['info']['source'] = source_flag
+            new_torrent.metainfo['comment'] = 'Uploaded to ' + tracker
```

Comment:
This is where you want to update this https://github.com/Zips-sipZ/Upload-Assistant/blob/1d1c02d9e60224ce1397d35bc49ba2440d4e3f56/src/prep.py#L2530

It should retain the created by comment, since clients would not otherwise show this. Clients do have the tracker URL as a UI option, so the tracker should just be appended to the existing comment, or changed in some way to retain the original comment. Thanks.

Reply:
What is the added value of the original comment? You already have the created by field which still holds the same information. Do let me know if there is a specific reason why it is what it is.

I've tested this with my client (qbit) and it shows this just fine, I can replace the tracker with trackerurl.

Reply:
> Editing that option in prep wouldn't make it unique to that one torrent being injected in your client, it would just edit the BASE (template) for all. Which is why I didn't edit that one because it's replaced by this anyway.

The BASE torrent is the source of the torrent file being modified in common.edit_torrent

``` python
    async def edit_torrent(self, meta, tracker, source_flag, torrent_filename="BASE"):
        if os.path.exists(f"{meta['base_dir']}/tmp/{meta['uuid']}/{torrent_filename}.torrent"):
```
``` python
    async def add_tracker_torrent(self, meta, tracker, source_flag, new_tracker, comment):
        if os.path.exists(f"{meta['base_dir']}/tmp/{meta['uuid']}/BASE.torrent"):
            new_torrent = Torrent.read(f"{meta['base_dir']}/tmp/{meta['uuid']}/BASE.torrent")
```
Hence editing the BASE torrent is the most logical place here, since it sets the original values, and is the most logical place to catch any edge cases.

The fields in prep.create_torrent should be modified also:
https://github.com/Audionut/Upload-Assistant/blob/8f2790558d212d003d8c9cd1322a78625d8f73b3/src/prep.py#L2509

As this will catch edge cases such as torrent regeneration in [ANT.py](https://github.com/Audionut/Upload-Assistant/blob/8f2790558d212d003d8c9cd1322a78625d8f73b3/src/trackers/ANT.py#L108).

I assume the function was added to common.py as an easy method to just copy the base torrent with modified values.

The `created_by` field is only used to track what created the torrent file (ie: sysops/devs), the `comment` field is the general user feedback field displayed in client. Almost every site would sanatize the comment field to minimise potential abuse, so it's only really function is feedback to the creator. Personally I just use the tracker column to track torrents, but if you wish the tracker name or url to be added here, that's fine, but I wish the original comment to remain in place also.

What would be really nice is requesting the actual the torrent page link (attributes.details_link in UNIT3D) from the site API's and injecting _that_ into the comment field before injecting the torrent file into the client, so then you have an actual direct link to the torrent on site.